src/app/legacy/containers/SocialEmbed/common/styles.js

```diff
@@ -16,7 +16,8 @@ const getWrapperHeightStyles = oEmbed => {
    * wrappers. This helps reduce layout shift. It is not applied to fallbacks.
    */
   const MIN_HEIGHT = '14rem';
-  if (oEmbed?.height) return `min-height: ${oEmbed.height / 16}rem`;
+  if (oEmbed?.height && oEmbed?.height !== '100%')
+    return `min-height: ${oEmbed.height / 16}rem`;
```

Comment:
Really struggled to read this without braces, maybe worth adding them for these if statements?

Reply:
Yea I agree, I'll add those.